projects/woff2/Readme.md

```diff
@@ -24,15 +24,23 @@ Unzip Sydr (`sydr.zip`) in `projects/woff2` directory:
 
     # sydr-fuzz -c convert_woff2ttf_fuzzing.toml run
 
+## Minimize corpus:
+
+    # sydr-fuzz -c convert_woff2ttf_fuzzing.toml cmin
+
+## Check security predicates:
+
+    # sydr-fuzz -c convert_woff2ttf_fuzzing.toml security
+
+## Crash triage with CASR:
+
+    # sydr-fuzz -c convert_woff2ttf_fuzzing.toml casr
+
 ## Get LCOV HTML coverage report:
 
     # sydr-fuzz -c convert_woff2ttf_fuzzing.toml cov-export -- -format=lcov > convert_woff2ttf_fuzzing.lcov
```

Comment:
Remove these two commands and add one command of new format

Reply:
Done